pipert2/utils/queue_wrapper.py

```diff
@@ -84,7 +87,13 @@ def queue_worker(self):
 
         for item in iter(self.mp_queue.get, None):
             try:
-                self.out_queue.put(item, block=True, timeout=1)
+                message = Message.decode(item)
+
+                if callable(self.receive):
+                    received_data = self.receive(message.payload.data)
+                    message.update_data(received_data)
```

Comment:
I didn't understand why do we need this, Maybe add a comment to explain this section or extract a method (the better option) with reflected name & docs.